pages/common/crane.md

```diff
@@ -4,26 +4,34 @@
 > Some subcommands such as `pull`, `push`, `copy`, etc. have their own usage documentation.
 > More information: <https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane.md/>.
 
-- Execute a `crane` subcommand:
+- Build or pull images if they don't exists:
 
-`crane {{subcommand}}`
+`crane up/lift {{[-d|--detach]}}`
```

Comment:
Is this an actual subcommand or is up and lift separate subcommands that do the same thing?

If yes then we can update it to:


```suggestion
`crane {{up|lift}} {{[-d|--detach]}}`
```